tests/unit/copydb.sh

```diff
@@ -27,10 +27,8 @@ create collation if not exists mycol
  );
 EOF
 
-pgcopydb list collations --source "${PGCOPYDB_SOURCE_PGURI}"
-
 # pgcopydb fork uses the environment variables
-pgcopydb fork --skip-collations --debug
+pgcopydb clone --skip-collations --debug
```

Comment:
Using the `pgcopydb fork` variant of the command was done on purpose here, to have some unit testing exercise that spelling of the command...